### PR TITLE
ci: Enable semantic PRs

### PR DESCRIPTION
**Pull request by Xuanwo**
Signed-off-by: Xuanwo <github@xuanwo.io>

I hereby agree to the terms of the CLA available at: https://databend.rs/dev/policies/cla/

## Summary

ci: Enable semantic PRs

Part of https://github.com/datafuselabs/databend/issues/6384

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -3,18 +3,3 @@ I hereby agree to the terms of the CLA available at: https://databend.rs/dev/pol
...
-- Bug Fix
-- Improvement
-- Performance Improvement
-- Build/Testing/CI
-- Documentation
-- Not for changelog (changelog entry is not required)
-
-## Related Issues
-
-Fixes #issue
```
Comment (PsiACE) [inline .github/PULL_REQUEST_TEMPLATE.md:19]: I think it can be kept.

Comment (Xuanwo) [reply .github/PULL_REQUEST_TEMPLATE.md:19]: Don't make sense to me, how about adding under summary?

**Review by PsiACE — APPROVED** [review]

**Comment by BohuTANG** [comment]
The contribution document need modify https://databend.rs/doc/contributing/good-pr

**Comment by Xuanwo** [comment]
> The contribution document need modify https://databend.rs/doc/contributing/good-pr

Addressed, PTAL

**Review by BohuTANG — APPROVED** [review]

**Pull request merged**